skt/mls.py

```diff
@@ -322,3 +323,31 @@ def get_meta_table_item(meta_table: str, item_name: str, aws_env: AWSENV = AWSEN
         raise MLSModelError(response.get("error"))
     else:
         return results
+
+
+def meta_to_pandas(meta_table: str, aws_env: AWSENV = AWSENV.STG.value) -> Any:
+    """
+    Get a meta_table as pandas dataframe
+    Args. :
+        - meta_table   :   (str) the name of meta_table
+        - aws_env      :   (str) AWS ENV in 'stg / prd / dev' (default is 'stg')
+    Returns :
+        - A Pandas dataframe type of the item_meta
+    """
+    assert type(meta_table) == str
+    assert type(aws_env) == str
+
+    url = AB_URL_PREFIX
+    if aws_env in (AWSENV.STG.value, AWSENV.DEV.value):
+        url = f"{url}.{aws_env}"
+    url = f"{url}.sktmls.com{MLS_META_API_URL}/{meta_table}"
+
+    response = requests.get(url).json()
+
+    items = response["results"]["items"]
+    key = pd.DataFrame.from_records(items)["name"]
+    values = pd.DataFrame.from_records(pd.DataFrame.from_records(items)["values"])
```

Comment:
meta_table에서 id는 필요없나요? 잘 몰라서 ㅋㅋ 필요없으면 이 코멘트는 그냥 무시하셔도 될듯요
```suggestion
    key = pd.DataFrame.from_records(items)["name"]
    index = pd.DataFrame.from_records(items)["id"]
    values = pd.DataFrame.from_records(pd.DataFrame.from_records(items)["values"])
```

Reply:
네 id는 필요없습니다 ㅋㅋ